lib/rdf/ldp/helper.rb

```diff
@@ -3,14 +3,16 @@
 # mixin methods
 module RDF
   class LDP < RDF::StrictVocabulary("http://www.w3.org/ns/ldp")
+    extend Deprecation
```

Comment:
This is new to me, looks interesting.  I'm wondering whether this should be `include` vs. `extend`.  See 
http://stackoverflow.com/questions/156362/what-is-the-difference-between-include-and-extend-in-ruby